en/docs/fhir/search/search-parameters.md

```
@@ -0,0 +1,178 @@
+GET /fhir/r4/Patient?mrn=12345
+```
+
+## Common resource parameters (reference)
+
+These are standard FHIR search parameters clients commonly use. When generated from an IG, matching entries already appear in `api_config.bal`; implement the search logic for the parameters your deployment supports.
+
+### Patient
+
+| Parameter | Type | Example |
+|-----------|------|---------|
+| `identifier` | token | `?identifier=http://example.org/mrn|12345` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)